tests/functional/fuzzy/test_fuzzy.py

```diff
@@ -98,14 +112,195 @@ def test_fuzz_config():
     detect_crashes(output.stdout)
 
 
-# TODO: use as AFL plugin which handles starting, stopping and logging
-def wait_for_completion(fuzzing_time_seconds: int, repo_path: str, ):
+def test_fuzz_trace_file():
+    fuzzing_time_seconds = 10 * 60
+    iotrace: IotracePlugin = TestRun.plugins['iotrace']
+
+    repo_path: str = f"{iotrace.working_dir}/slit-afl"
+
+    # Make sure AFL is installed
+    if not is_afl_installed():
+        install_afl()
```

Comment:
Shouldn't this be part of dependencies?

Reply:
I think it's better not to add them to dependencies since fuzzy test will not always be run with other tests due to how long they run.

Additionally there is no installer for AFL so automatically installing it here should make the setup process easier.